client/FxAccountClient.js

```diff
@@ -94,6 +95,30 @@ define([
           var requestOpts = {};
 
           if (options) {
+            if (options.device) {
+              requestOpts.device = {};
```

Comment:
Looks like options.device is not documented in the doc string.
It is not part of the 
```
    *   @param {String} [options.resume]
    *   Opaque url-encoded string that will be included in the verification link
    *   as a querystring parameter, useful for continuing an OAuth flow for
    *   example.
    *   @param {String} [options.lang]
``` 

doc blob